src/queries/queries.types.ts

```diff
@@ -67,6 +68,7 @@ export type QueryArrayVariable = {
   items: {
     type: "string" | "number" | "boolean" | "date";
   };
+  optional?: boolean;
```

Comment:
There are quiet a few `description: """` ... I wonder if it's worth making 'description' optional as well?